docs/src/modules/components/ThemeViewer.tsx

```diff
@@ -146,6 +152,9 @@ function ObjectEntry(props: { itemId: string; objectKey: string; objectValue: an
 
   return (
     <TreeItem
+      sx={{
+        paddingLeft: `${depth * 8}px`,
+      }}
```

Comment:
The default value of `spacing` in theme is 8, so the value passed to the spacing methods in the `sx` prop are multiplied by 8. So this should work:
```suggestion
      sx={{
        paddingLeft: depth,
      }}
```